### ISSUE TITLE
L'install via npm install alsactreations/bretzel ne fonctionne pas

### ISSUE THREAD

**Issue by PhilippeVay** [alsacreations/bretzel#61]
[Commit](https://github.com/alsacreations/bretzel/commit/7f78f5f5892912de661efd52b40642006cb8d8ca) de @raphaelgoetter dans la doc.

**Comment by raphaelgoetter** [comment]
oui, on a testé et on confirme. C'est ok avec un git clone mais pas avec un npm install.

**Comment by PhilippeVay** [comment]
```
$ node -v && npm -v
v8.9.1
6.1.0
```

Même souci avec KNACSS. Si je clone KNACSS ou bretzel et `npm i` l'autre, j'obtiens bien les 2 installés comme projet principal d'une part et dépendance d'autre part.

Ce qui fonctionne : 

```
npm init
(plein de questions pour la création d'un projet)
npm i bretzel
```

Ce qui fonctionne aussi : 

```
bretzel-start -c
bretzel-start <nom_du_projet>
cd <nom_du_projet>
npm i
```

Conclusion : il faut déjà avoir initialisé un projet avec **`npm init`** avant de pouvoir y ajouter des dépendances.

(je fermerai le sujet jeudi 16)
